### PR TITLE
add docs for exception handler

### PR DESCRIPTION
**Pull request by Lendemor**

### CODE REVIEW

File: docs/utility_methods/exception_handlers.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
+# Exception handlers
+
+_Added in v0.5.7_
+
+Exceptions handlers are function that can be assigned to your app to handle exceptions that occur during the application runtime.
```
Comment (picklelo) [inline docs/utility_methods/exception_handlers.md:5]: function -> functions

File: docs/utility_methods/exception_handlers.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+They are useful for customizing the response when an error occurs, logging errors, and performing cleanup tasks.
+
+## Types
+
+Reflex support two type of exception handler `frontend_exception_handler` and `backend_exception_handler`.
```
Comment (picklelo) [inline docs/utility_methods/exception_handlers.md:10]: handler -> handlers

File: docs/utility_methods/exception_handlers.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+
+They are both used to handle exceptions that occur in the `frontend` and `backend` respectively.
```
Comment (picklelo) [inline docs/utility_methods/exception_handlers.md:12]: remove "both". Maybe explain a bit what a frontend error is compared to a backend. A frontend is a Javascript error, while a backend is an error in the event handler

File: docs/utility_methods/exception_handlers.md [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+```md alert warning
+# Only named functions are supported as exception handler.
+```
+
+## Examples
+
+```python
+import reflex as rx
+
+def custom_frontend_handler(exception: Exception) -> None:
```
Comment (picklelo) [inline docs/utility_methods/exception_handlers.md:29]: Let's add some examples of actual handlers here to show how it could work

**Review by picklelo — COMMENTED** [review]
Looks good, some minor nits.

I'm wondering though (outside of this PR) should exception handlers be tied to event handlers? then for example you could update your state if something happened.

File: docs/utility_methods/exception_handlers.md (hunk 4 above)
Comment (Lendemor) [reply docs/utility_methods/exception_handlers.md:31]: What do you have in mind for actual handlers? Printing the exception ?

Comment (picklelo) [reply docs/utility_methods/exception_handlers.md:31]: Yeah I think printing it makes sense, just to show a practical way someone may use it

File: docs/utility_methods/exception_handlers.md [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+    # My custom logic for frontend errors
+    pass
+
+def custom_backend_handler(exception: Exception) -> rx.event.EventSpec:
+    # My custom logic for backend errors
+    pass
+
+app = rx.App()
```
Comment (picklelo) [inline docs/utility_methods/exception_handlers.md:39]: Try this with the kwargs approach instead

**Review by picklelo — APPROVED** [review]

**Pull request merged**